this doesn't do anything, you're not meant to use the internal deps
--
--- a/Makefile.am
+++ b/Makefile.am
@@ -9,7 +9,6 @@
 COMMON_LDFLAGS =  -Wl,-z,relro,-z,now
 
 libtool: $(LIBTOOL_DEPS)
-	$(SHELL) ./config.status libtool
 
 noinst_PROGRAMS = lib/doc/bin/docproc
 
